### PR TITLE
Migrating references to yum::epel to new yum-epel cookbook

### PR DESCRIPTION
**Pull request by noconnor**
Fixing issue with yum::epel cookbook include.
Looks like there was a major upstream release of the yum cookbook (3.X) that moved yum::epel recipe to a new yum-epel cookbook.

Se discussion here: https://github.com/bflad/chef-skyline/issues/1

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 license           'Apache 2.0'
 description       'Installs/Configures Protocol Buffer'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '1.0.1'
+version           '1.0.2'
```
Comment (bflad) [inline metadata.rb:7]: Most maintainers I've dealt with prefer if they manage the versioning themselves (including myself).

**Comment by bflad** [comment]
I'll get this merged in later tonight and released as 1.1.0. Thanks.

**Comment by noconnor** [comment]
Great thanks

**Comment by kevinmeredith** [comment]
I cloned my local repo, which has a Vagrantfile & Berksfile in it, and then ran `vagrant up --provision`, yet I saw the above error:

```
Chef::Exceptions::RecipeNotFound
--------------------------------
could not find recipe epel for cookbook yum
```

I'm going to try the workarounds provided in @noconnor's message above. Should this problem have been fixed with a commit from @bflad on April 7? 

Thanks

**Comment by noconnor** [comment]
Looks like the berksfile is trying to pull the protobuf cookbook from the opscode site and I don't think the version containing the fix (1.1.0) has been published there yet. Maybe try updating the berksfile to pull the protobuf dependency from github instead of opscode

**Comment by kevinmeredith** [comment]
Thanks, @noconnor.

For my own understanding, could you please say more as to how you know that:

`Looks like the berksfile is trying to pull the protobuf cookbook from the opscode site`.

Also, does 

`Maybe try updating the berksfile to pull the protobuf dependency from github instead of opscode`

mean this in my `Berksfile`:

`cookbook 'yum', git: 'https://github.com/opscode-cookbooks/yum.git'`?

**Comment by noconnor** [comment]
Try something like: 
cookbook 'protobuf', '1.1.0', github: 'bflad/chef-protobuf'

**Comment by kevinmeredith** [comment]
@noconnor, thanks for that suggestion. I added the following to my `Berksfile`:

`cookbook 'protobuf', '= 1.1.0', git: 'https://github.com/bflad/chef-protobuf.git'`

but I got the same error:

```
Chef::Exceptions::RecipeNotFound
--------------------------------
could not find recipe epel for cookbook yum
```

**Comment by noconnor** [comment]
Try running: berks update

**Comment by kevinmeredith** [comment]
I don't have `berks` installed, but I did destroy my vagrant box, and then delete the `Berksfile.lock` file.

Then I ran `vagrant up --provision`.

I updated my `Berksfile` to specify a 2.x version of the yum cookbook:

`cookbook 'yum', '= 2.4.4' #git: 'https://github.com/opscode-cookbooks/yum.git'`

as per the github warning under the "Notes" section: https://github.com/opscode-cookbooks/yum.

But I'm getting this error now:

`No version specified, and no candidate version available for kernel-devel`

I'm not sure if this is related to the original problem that I had though.

**Comment by kevinmeredith** [comment]
I posted this stack overflow question

http://stackoverflow.com/q/22988922/409976

**Comment by bflad** [comment]
Sorry about that - I have uploaded protobuf 1.1.0 to the community site.

**Comment by kevinmeredith** [comment]
Hi @bflad. After cloning my repo into a new directory, I ran `vagrant up --provision`, yet I'm still getting this error:

```
Chef::Exceptions::Package
-------------------------
No version specified, and no candidate version available for kernel-devel
```

As I understand your response, you would've expected your push to have fixed my issue? 

Here's my recipes that I'm using:

```
cookbook 'yum'
cookbook 'make_custom_user', path: 'cookbooks\make_custom_user'
cookbook 'make_custom_directory', path: 'cookbooks\make_custom_directory'
cookbook 'java_custom', path: 'cookbooks\java_custom'
cookbook 'maven_custom', path: 'cookbooks\maven_custom'
cookbook 'tomcat_custom', path: 'cookbooks\tomcat_custom'
cookbook 'git'
cookbook 'ant'
cookbook 'gradle_custom', path: 'cookbooks\gradle_custom'
cookbook 'set_env_var_custom', path: 'cookbooks\set_env_var_custom'
cookbook 'nodejs
```

Thanks.

**Pull request merged**